MonitorControl/AppDelegate.swift

```diff
@@ -166,27 +161,39 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }
   }
 
-  private func setupLayout() {
+  private func setupViewControllers() {
     let storyboard: NSStoryboard = NSStoryboard(name: "Main", bundle: Bundle.main)
+    let mainPrefsVc = storyboard.instantiateController(withIdentifier: "MainPrefsVC")
+    let keyPrefsVc = storyboard.instantiateController(withIdentifier: "KeysPrefsVC")
+    let displayPrefsVc = storyboard.instantiateController(withIdentifier: "DisplayPrefsVC")
+    let advancedPrefsVc = storyboard.instantiateController(withIdentifier: "AdvancedPrefsVC")
     let views = [
-      storyboard.instantiateController(withIdentifier: "MainPrefsVC"),
-      storyboard.instantiateController(withIdentifier: "KeysPrefsVC"),
-      storyboard.instantiateController(withIdentifier: "DisplayPrefsVC"),
+      mainPrefsVc,
+      keyPrefsVc,
+      displayPrefsVc,
+      advancedPrefsVc,
     ]
     prefsController = MASPreferencesWindowController(viewControllers: views, title: NSLocalizedString("Preferences", comment: "Shown in Preferences window"))
+    if let displayPrefs = displayPrefsVc as? DisplayPrefsViewController {
+      displayPrefs.displayManager = self.displayManager
+    }
+    if let advancedPrefs = advancedPrefsVc as? AdvancedPrefsViewController {
+      advancedPrefs.displayManager = self.displayManager
+    }
   }
 
   private func subscribeEventListeners() {
     // subscribe KeyTap event listener
     NotificationCenter.default.addObserver(self, selector: #selector(handleListenForChanged), name: NSNotification.Name(Utils.PrefKeys.listenFor.rawValue), object: nil)
     NotificationCenter.default.addObserver(self, selector: #selector(handleShowContrastChanged), name: NSNotification.Name(Utils.PrefKeys.showContrast.rawValue), object: nil)
     NotificationCenter.default.addObserver(self, selector: #selector(handleFriendlyNameChanged), name: NSNotification.Name(Utils.PrefKeys.friendlyName.rawValue), object: nil)
+    NotificationCenter.default.addObserver(self, selector: #selector(self.handlePreferenceReset), name: NSNotification.Name(Utils.PrefKeys.preferenceReset.rawValue), object: nil)
 
```

Comment:
Should we replace all notification name init and make an extension for `NSNotification.Name` like so ?

```swift
extension NSNotification.Name {
    static let listenFor = Notification.Name.init(Utils.PrefKeys.listenFor.rawValue)
    …
}
```